README.md

```diff
@@ -84,16 +85,17 @@ WARNING: headers exchange binding not supported by library helpers functions
 config :myapp,
   consumers: [
     %{
-      handler_module: Your.Handler.Module,
+      handler_module: Myapp.Consumer,
       prefetch_count: 100,
       backoff: 10_000
     }
+  ]
 
-config :myapp, Your.Handler.Module, %{
+config :myapp, Myapp.Consumer, %{
     queue: "my_queue",
     exchanges: [
       %{name: "amq.topic", type: :topic, routing_keys: ["my.routing_key1","my.routing_key2"], opts: [durable: true]},
-      %{name: "my_exchange", type: :direct, routing_keys: ["my_queue"]},
+      %{name: "my_exchange", type: :direct, routing_keys: ["my_queue"], opts: [durable: true]},
```

Comment:
Our typo :smile: 
```suggestion
      %{name: "my_exchange", type: :direct, routing_keys: ["my.direct_routing_key"], opts: [durable: true]},
```

Reply:
This is ok, is a direct exchange, so routing key and queue name must match